Formula/stubby.rb

```diff
@@ -0,0 +1,65 @@
+class Stubby < Formula
+  desc "DNS privacy enabled stub resolver service based on getdns"
+  homepage "https://github.com/saradickinson/stubby"
+  url "https://github.com/saradickinson/stubby/archive/0.1.1.tar.gz"
+  sha256 "8a4592b44eea1da24e5387632ad2546e428a7228600ffb712414fa0f2a4e4919"
+
+  head do
+    url "https://github.com/saradickinson/stubby.git", :branch => "update_readme"
```

Comment:
This should be
```
  head "https://github.com/saradickinson/stubby.git", :branch => "update_readme"
```

But that doesn't strike me as a branch name that's likely to be around for very long. It might be best to just leave out head altogether for this formula.